ansible/roles/docker/tasks/main.yml

```diff
@@ -69,8 +69,8 @@
   tags: configure
 
 - name: Add any insecure registrys to docker config
-  lineinfile: dest={{ docker_config_dir }}/docker regexp=^INSECURE_REGISTRY= line=INSECURE_REGISTRY="{% for reg in insecure_registrys %}--insecure-registry={{ reg }} {% endfor %}"
-  when: insecure_registrys is defined and insecure_registrys > 0
+  lineinfile: dest={{ docker_config_dir }}/docker regexp="^INSECURE_REGISTRY=" line="INSECURE_REGISTRY={% for reg in insecure_registrys %}--insecure-registry={{ reg }} {% endfor %}"
```

Comment:
I agree this would be more clear. This does not conform to the other `lineinfile`s in the `main.yml`.

I tend to write my ansible as you have.

I can do this for all of them or just this one. If figured such a change would be a separate issue and pull request (happy to open and fix).